packages/noir-ethereum-api/src/noir/oracles.ts

```diff
@@ -7,12 +7,36 @@ import { encodeField } from './encode.js';
 
 export type Oracles = (name: string, args: string[][]) => Promise<ForeignCallOutput[]>;
 
+export interface AccountWithProof {
+  balance: string,
+  codeHash: string,
+  nonce: string,
+  stateRoot: string[],
+  key: string[],
+  value: string[],
+  proof: string[],
+  depth: string,
+}
+
+export function serializeAccountWithProof(account: AccountWithProof): ForeignCallOutput[] {
+  return [
+    account.balance,
+    account.codeHash,
+    account.nonce,
+    account.stateRoot,
+    account.key,
+    account.value,
+    account.proof,
+    account.depth
+  ];
+}
+
```

Comment:
btw is it time to extract account oracle to separate file?
(header oracle coming soon)